addon/serializer.js

```diff
@@ -27,10 +27,21 @@ import Ember from 'ember';
 export default Ember.Object.extend({
 
   /**
-    The `store` property is the application's `store` that contains all records.
-    It's injected as a service.
-    It can be used to push records from a non flat data structure server
-    response.
+    The `store` property is the application's `store` that contains
+    all records.  It can be used to look up serializer for other model
```

Comment:
Extreme Nitpicking™️ – double space here :)

Also `look up serializer for other model type` => `look up serializers for other model types` ?